MovieApp/Views/MoviesCell.m

```diff
@@ -24,6 +24,13 @@ - (void)awakeFromNib {
     [_title setUserInteractionEnabled:NO];
     [_title setTextAlignment:NSTextAlignmentLeft];
     [_title setBaselineAdjustment:UIBaselineAdjustmentAlignBaselines];
+    CAGradientLayer *gradientMask = [CAGradientLayer layer];
```

Comment:
I suppose you will use this gradient layer in multiple cases, so create some Customization class that will contain this and other methods for customatization.